### PR TITLE
EventNotify updates 

### PR DESCRIPTION
**Pull request by Brackleycassinga**
- Added actionable and resourceId fields to notifications for node configuration to a site.
- Updated eventNotify to save resourceId and isActionable information to the notification database.
- Modified the GraphQL schema in the BFF to include actionable and resourceId.
- Introduced new events: siteCreate and nodeStateChange.
- Enhanced the event listener in EventNotify to capture nodeStateChange events.
- Added a configNode action button in the notification popover when it is actionable, enabling users to configure the node to the site.

### CODE REVIEW

File: interfaces/console/src/app/configure/page.tsx [hunk 1]
```diff
@@ -38,10 +38,12 @@ const Page = () => {
     setIsChecked(isChecked);
   };
   const onInstallProgressComplete = () => {
-    const id = 'uk-sa9001-tnode-a1-1234';
-    router.push(`/configure/node/${id}?lat=-4.322447&lng=15.307045`);
-  };
+    const id = searchParams.get('nodeId') ?? 'uk-sa9001-tnode-a1-1234';
+    const lat = searchParams.get('lat') ?? '-4.322447';
+    const lng = searchParams.get('lng') ?? '15.307045';
```
Comment (salman-codewar) [inline interfaces/console/src/app/configure/page.tsx:43]: We need to remove dummy nodeId and lat, lng.  If any of these param missing we can't continue with the configure flow. We can just show alert snackbar.

File: interfaces/console/src/app/console/layout.tsx [hunk 2]
```diff
@@ -46,9 +48,9 @@ export default function ConosleLayout({
     setSnackbarMessage,
     subscriptionClient,
   } = useAppContext();
-  const [notifications, setNotifications] = useState<
-    NotificationsResDto[] | []
-  >([]);
+  const router = useRouter();
+  const [notifications, setNotifications] = useState<any>([]);
+
```
Comment (salman-codewar) [inline interfaces/console/src/app/console/layout.tsx:53]: Why we change `notifications` type define from `NotificationsResDto[]` to `any`?

File: interfaces/console/src/app/console/layout.tsx [hunk 3]
```diff
@@ -101,6 +103,16 @@ export default function ConosleLayout({
...
+      /**
+       * Sets the notifications from the response data
+       * @example
+       * setNotificationsResponse(response)
+       * undefined
+       * @param {Object} res - The response object containing notification data.
+       * @returns {void} No return value.
+       * @description
+       *   - Extracts notifications from the response object safely using optional chaining.
+       */
```
Comment (salman-codewar) [inline interfaces/console/src/app/console/layout.tsx:115]: Is this for future reference?

**Review by salman-codewar — CHANGES_REQUESTED** [review]

File: interfaces/console/src/app/console/layout.tsx (hunk 2 above)
Comment (Brackleycassinga) [reply interfaces/console/src/app/console/layout.tsx]: Fixed

**Pull request closed without merging**